<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandleInfoClass</class>
 <widget class="QDialog" name="HandleInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HandleInfoClass</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
   /* Темный фон */
    color: #dcdcdc; /* Светлый текст */
    font-family: &quot;Arial&quot;, sans-serif;
}

QLineEdit {
    background-color: #2d2d2d; /* Темный фон для полей ввода */
    border: 1px solid #4a4a4a; /* Тонкая граница */
    border-radius: 5px; /* Закругленные углы */
    padding: 5px;
    color: #dcdcdc; /* Светлый цвет текста */
}

QLineEdit:focus {
    border: 1px solid #6a5acd; /* При фокусе — светло-фиолетовая граница */
    background-color: #353535; /* Слегка светлее фон при фокусе */
}

QPushButton {
    background-color: #4a4a4a; /* Темно-серый фон кнопок */
    border: 1px solid black; /* Граница с фиолетовым оттенком */
    border-radius: 5px; /* Закругленные углы кнопок */
    padding: 5px;
    color: #dcdcdc; /* Светлый текст на кнопках */
    font-size: 14px;
}

QPushButton:hover {
    border-color: #8a7fb3; /* Светлая граница */
}

QPushButton:pressed {
    background-color: #5a4f85; /* Темный фиолетовый при нажатии */
}

QLabel {
    font-size: 14px;
    font-weight: bold;
    margin-bottom: 5px;
}

QFrame {
    border: none;
}

QGroupBox {
    border: 1px solid #3a3a3a;
    border-radius: 5px;
    padding: 10px;
    background-color: #2d2d2d;
}

QGroupBox:title {
    color: #dcdcdc;
    padding: 5px;
    font-size: 18px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Change Information</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>80</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit">
       <property name="placeholderText">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="surnameEdit">
       <property name="placeholderText">
        <string>Surname</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>The unchanged data will remain the same.

</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="codeLabel">
     <property name="text">
      <string>Enter code from message to your email</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="codeLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkCodeButton">
       <property name="text">
        <string>Save information</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="backToMenuButton">
     <property name="text">
      <string>Return </string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
